Task_Array/index.js

```diff
@@ -1,67 +1,68 @@
-let arr = [1, 3, 2, 5, 4];
-
-// Filter the odd number using the filter function of the array and store it in filterArr declared below.
-
-let filterArr;
-//Write your code below
-
-//Write your code above
-console.log("Filtered Array:-", filterArr);
-
-// Find the square of each number of the array using map fumction and store it in squaredArray variable declared below. Output should be [1,9,4,16]
-let squaredArray;
-//Write your code below
```

Comment:
The variable 'squaredArray' is declared twice. Remove line 13 to avoid redeclaration.